Dockerfile

```
@@ -2,10 +2,12 @@ FROM python:3.13-alpine
 
 WORKDIR /usr/src/app
 
-RUN apk add --no-cache python3 py3-pip ca-certificates tzdata
+RUN apk add --no-cache python3 py3-pip ca-certificates tzdata tini
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)